lfs/util.go

```diff
@@ -115,6 +115,11 @@ func FilenamePassesIncludeExcludeFilter(filename string, includePaths, excludePa
 	if len(includePaths) > 0 {
 		matched := false
 		for _, inc := range includePaths {
+			// Special case local dir, matches all (inc subpaths)
+			if inc == "." || inc == "./" || inc == ".\\" {
+				matched = true
+				break
+			}
```

Comment:
Doh, I meant a `map[string]bool`, not a slice. But yours is fine. I'd just move it outside the func so that it doesn't get created every time `FilenamePassesIncludeExcludeFilter()` is called.